app/src/main/res/layout/activity_preg.xml

```diff
@@ -167,6 +167,16 @@
                     android:textColor="@color/black"
                     android:textColorHint="@color/black" />
 
+                <TextView
+                    android:id="@+id/registerErrorText"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:text="Register Error Message"
+                    android:textAlignment="center"
+                    android:textColor="@color/black"
+                    android:textSize="18sp"
+                    android:visibility="gone" />
+
 
```

Comment:
Hmm I think just saying register error would be bit vague for the  person who is registering. Maybe like at each required input there could be error message saying the input is empty or the password and confirm password does not match.